Formula/lib/libvirt-python.rb

```diff
@@ -0,0 +1,41 @@
+class LibvirtPython < Formula
+  desc "Libvirt virtualization API python binding"
+  homepage "https://www.libvirt.org/"
+  url "https://download.libvirt.org/python/libvirt-python-10.3.0.tar.gz"
+  sha256 "0333781ffef915d984f36a9b475ae8df6d01763883eefbd138d14c7591f51f2f"
+  license "LGPL-2.1-or-later"
+
+  livecheck do
+    url "https://download.libvirt.org/python/"
+    regex(/href=.*?libvirt-python[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.12" => [:build, :test]
+  depends_on "libvirt"
+
+  def pythons
+    deps.map(&:to_formula)
+        .select { |f| f.name.match?(/^python@\d\.\d+$/) }
+        .map { |f| f.opt_libexec/"bin/python" }
+  end
+
+  def install
+    pythons.each do |python|
+      system python, "-m", "pip", "install", *std_pip_args, "."
+    end
+  end
+
+  test do
+    system "python3", "-c",
+           # language=Python
```

Comment:
not sure what is it for?
```suggestion
```

Reply:
Such a comment just enables [language injection](https://www.jetbrains.com/help/idea/using-language-injections.html) in IntelliJ IDEA which I use.
It doesn't hurt functionally, but enables static analysis and autocompletion within the script in the string literal.
It looks like this
<img width="669" alt="Screenshot 2024-05-24 at 07 15 51" src="https://github.com/Homebrew/homebrew-core/assets/7255667/a3d93b4a-85da-4949-85bd-97d0d5504b8d">